rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/mapper/MapperEntityToResponse.kt

```diff
@@ -0,0 +1,29 @@
+package systems.ajax.englishstudytelegrambot.nats.mapper
+
+import entity.LibraryOuterClass
+import entity.WordOuterClass
+import systems.ajax.englishstudytelegrambot.entity.Library
+import systems.ajax.englishstudytelegrambot.entity.Word
+
+fun Library.toLibraryResponse(): LibraryOuterClass.Library =
+    LibraryOuterClass.Library.newBuilder()
+        .setId(id.toHexString())
+        .setName(name)
+        .setOwnerId(ownerId)
+        .build()
+
+fun Word.toWordResponse(): WordOuterClass.Word =
+    WordOuterClass.Word.newBuilder()
+        .setId(id.toHexString())
```

Comment:
Not sure but be careful with mapping if it can be null. If it is like I think use **takeIf(hasId())** or something similar.